pkg/cgroups/cgroups.go

```diff
@@ -1,26 +1,27 @@
 package cgroups
 
 import (
-	"bufio"
 	"fmt"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strconv"
-	"strings"
 
+	"github.com/square/p2/pkg/types"
 	"github.com/square/p2/pkg/util"
 )
 
 // maps cgroup subsystems to their respective paths
 type Subsystems struct {
 	CPU    string
 	Memory string
+	Prefix string
 }
 
-var Default Subsystems = Subsystems{
+var DefaultSubsystems = Subsystems{
```

Comment:
how did this ever work? parser bug?

Reply:
no parser bug... the name is `Default` and the type is `Subsystems`. Amazing.